MaintenanceManager/CHANGELOG.md

```diff
@@ -5,10 +5,14 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [1.0.1] - 2022-08-12
```

Comment:
Pls update the version for https://github.com/rdkcentral/rdkservices/pull/3015/files first since it needs to first go to main and OTA_5. So update it to 1.0.1 and bring it back to sprint/2208.

You can then update this change to 1.0.2